src/components/Form/Form.stories.mdx

```diff
@@ -1575,7 +1575,9 @@ This field implements the [**FieldDef**](#generic-field-props-fielddef) interfac
 | Name | Type | Default | Description |
 | ---- | ---- | --------| ----------- |
 | **`columns`** | `{ name: string; title: string }[]` | | Columns definition. |
-| **`columnTotalLabel`** | `string` | "Total" | optional - Default: "Total". Label shown on the column that displays the totals. |
+| **`columnTotalLabel`** | `string` | `true` | optional - Default: "Total". Label shown on the column that displays the totals. |
```

Comment:
I think there was a confusion when copying / pasting the lines here. columnTotalLabel says it's a string but it's default value is "true", and the displaySumRow says it's a boolean but it's default is "Total". Also all 3 descriptions are repeated.